includes/Helpers.php

```diff
@@ -16,65 +16,133 @@ class Helpers {
 	 * @author Allen Moore, 10up
 	 */
 	public function __construct() {
-		add_action( 'inline_svg', array( $this, 'inline_svg' ), 10, 1 );
+		add_action( 'inline_svg', array( $this, 'inline_svg' ), 10 );
 		add_action( 'svg_button', array( $this, 'svg_button' ), 10, 3 );
 	}
 
 	/**
-	 * Function that returns a SVG file path.
+	 * Function to return the full path to SVG assets.
+	 *
+	 * @author Allen Moore, 10up
+	 * @param  string $path a directory to append to the path.
+	 * @return string       the full path for assets.
+	 */
+	public function get_assets_path( $path = '' ) {
+
+		$full_path = get_svg_path_option();
+
+		/**
+		 * Filters the path.
+		 *
+		 * This function allows the path to include a custom directory
+		 * structure for use cases where SVG assets are stored in multiple
+		 * directories.
+		 *
+		 * @author Allen Moore, 10up
+		 * @param  string $full_path the path to assets from options.
+		 * @param  string $path      a directory to append to the path.
+		 */
+		$full_path = apply_filters( 'svg_path', $full_path, $path );
+
+		return $full_path;
+	}
+
+	/**
+	 * Function to detect the file path type.
+	 *
+	 * @author Allen Moore, 10up
+	 * @return Boolean returns true/false based on the file path type.
+	 */
+	public function get_path_type() {
+
+		$option_path = get_svg_path_option();
+
+		return (bool) filter_var( $option_path, FILTER_VALIDATE_URL	);
```

Comment:
It seems to me that this should be checking `$this->get_assets_path()` rather than `get_svg_path_option()`, since the filter could change a remote location to local or local to remote.